cmake_scripts/FindArrow.cmake

```diff
@@ -41,19 +41,35 @@ if (NOT ARROW_HOME)
     pkg_get_variable(ARROW_SO_VERSION arrow so_version)
     message(STATUS "Arrow SO version: ${ARROW_SO_VERSION}")
     set(ARROW_INCLUDE_DIR ${ARROW_INCLUDE_DIRS})
+    set(PYARROW_INCLUDE_DIR ${PYARROW_INCLUDE_DIR})
     set(ARROW_LIBS ${ARROW_LIBRARY_DIRS})
     set(ARROW_SEARCH_LIB_PATH ${ARROW_LIBRARY_DIRS})
   elseif(DEFINED ENV{VIRTUAL_ENV})
     find_path(ARROW_INCLUDE_DIR arrow/api.h HINTS
       $ENV{VIRTUAL_ENV}/lib/*/site-packages/pyarrow/include)
```

Comment:
We can use `PYARROW_INLCUDE_DIR` defined below for `ARROW_INCLUDE_DIR` too.

Reply:
remove the find_path call and run set(ARROW_INCLUDE_DIR ${PYARROW_INCLUDE_DIR}) after the get_include call

Reply:
Yes. I think so.

Reply:
I've made this change